cmd/config/init/init.go

```diff
@@ -19,98 +19,33 @@ func InitCmd() *cobra.Command {
 		Use:   "init <chain-id>",
 		Short: "Initialize a rollapp configuration on your local machine.",
 		Run: func(cmd *cobra.Command, args []string) {
-			rollappId := args[0]
-			denom := args[1]
-			home := cmd.Flag(FlagNames.Home).Value.String()
-			createLightNode := !cmd.Flags().Changed(lightNodeEndpointFlag)
-			rollappBinaryPath := getRollappBinaryPath(cmd)
-			decimals := getDecimals(cmd)
-			initConfig := InitConfig{
-				Home:              home,
-				RollappID:         rollappId,
-				RollappBinary:     rollappBinaryPath,
-				CreateDALightNode: createLightNode,
-				Denom:             denom,
-				HubID:             DefaultHubID,
-				Decimals:          decimals,
-			}
-
-			addresses := initializeKeys(initConfig)
-			if createLightNode {
-				if err := initializeLightNodeConfig(initConfig); err != nil {
-					panic(err)
+			initConfig := getInitConfig(cmd, args)
+			overwrite, err := prepareDirectory(initConfig.Home)
```

Comment:
naming convention confusing. `overwrite` is true though the `prepareDirectory` (which doesn't really prepare the directory if it's a fresh installation) is empty. so basically we're not overwriting anything.